packages/dhis2/README.md

```diff
@@ -14,6 +14,10 @@ core execute \
  -o ./tmp/output.json
 ```
 
+> **Important Note:**  
+> This adaptor uses the old tracker version. For the new version, use 6.0.
```

Comment:
Good thinking here! I hadn't thought of this.

Can we specify the range of API version numbers that will work, and maybe link to the DHIS2 docs?